code-challenges.test.js

```diff
@@ -12,34 +12,55 @@
 
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
 
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
 
 
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+// // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
 const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
+// // Expected output: "15 is divisible by three"
 const object2 =  { number: 0 }
-// Expected output: "0 is divisible by three"
+// // Expected output: "0 is divisible by three"
 const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
-
-
-// b) Create the function that makes the test pass.
-
-
-
+// // Expected output: "-7 is not divisible by three"
+
+// //First i will describe a function called divByThree, and have it take an object as an argument
+// // the exected output is a string that says if the number is divisible by three or not
+// // i will use modulo, conditionals. 
+
+describe("divByThree", () => {
+    it("returns whether the number inside is evenly divisible by three or not.", () => {
+        expect(divByThree(object1)).toEqual("15 is divisible by three");
+        expect(divByThree(object2)).toEqual("0 is divisible by three");
+        expect(divByThree(object3)).toEqual("-7 is divisible by three");
+    });
+  });
+
+// // b) Create the function that makes the test pass.
+
+const divByThree = (number) => {
```

Comment:
Yes you want the number. However, where is the value located for the number key? How do you get access to that value?